data/common/export.py

```diff
@@ -2,6 +2,8 @@
 import functools
 import json
 import logging
+from collections import OrderedDict
```

Comment:
actually I'm no more sure it's usefull to use an OrderedDict. We remove it?

Reply:
Im using to keep the same key order in the impacts object, is it bad ?